docs/data-structures.rst

```diff
@@ -98,19 +98,23 @@ Transaction Test (object)
 .. _transaction-results:
 
 Transaction Results (object)
---------------------------------
+----------------------------
 
 Transaction result equals to the result of the `Gavel <https://github.com/apiaryio/gavel.js>`__ validation library.
 
 -  valid (boolean) - Indicates whether the transaction is valid.
 -  fields (object)
-
-   - [fieldName: string]: :ref:`gavel-validator-output`
+   -  *uri* - :ref:`gavel-validation-result-field`
+   -  *method* - :ref:`gavel-validation-result-field`
+   -  *statusCode* - :ref:`gavel-validation-result-field`
+   -  *headers* - :ref:`gavel-validation-result-field`
+   -  *body* - :ref:`gavel-validation-result-field`
```

Comment:
Hmm, this way it is even better than what I thought originally!

```
- *body* - ...
```

in MSON means that there are properties with generic name, for example `body` (but it can be any string).


In this case, when we're listing all the properties, we don't need the `*` anymore. Also, the types could be expressed with parentheses:

```rst
   -  uri (:ref:`gavel-validation-result-field`)
   -  method (:ref:`gavel-validation-result-field`)
   -  statusCode (:ref:`gavel-validation-result-field`)
   -  headers (:ref:`gavel-validation-result-field`)
   -  body (:ref:`gavel-validation-result-field`)
```

Reply:
Got it! Updated here 26deef8.